install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
I vote for adding bash to FreeBSD install-deps.sh and changing all `#!/bin/bash` to `#!/usr/bin/env bash` in our scripts.

It looks like there is not a way to pass bash options when invoking via /usr/bin/env. Thus lines like this:
```
#!/bin/bash -e
```
have to be changed to
```
#!/usr/bin/env bash
set -e
```